gef-ui/server/utils/facility-helpers.js

```diff
@@ -20,13 +20,34 @@ const facilitiesChangedToIssuedAsArray = (application) => {
   return hasChanged;
 };
 
-const summaryIssuedChangedToIssued = (app, user, data) =>
-  ((app.status === CONSTANTS.DEAL_STATUS.UKEF_ACKNOWLEDGED) || (app.status === CONSTANTS.DEAL_STATUS.CHANGES_REQUIRED))
-   && user.roles.includes('maker') && data.details.canResubmitIssuedFacilities === true;
+const summaryIssuedChangedToIssued = (params) => {
+  const {
+    acceptableStatus,
+    acceptableRole,
+    app,
+    data,
+    user,
+  } = params;
```

Comment:
I think it should receive only the params it needs, not a blocker

Reply:
Already is.